exports.js

```diff
@@ -645,6 +645,7 @@ module.exports = {
         'blobSoftDeletionEnabled'       : require(__dirname + '/plugins/azure/storageaccounts/blobSoftDeletionEnabled.js'),
         'storageAccountsTlsVersion'     : require(__dirname + '/plugins/azure/storageaccounts/storageAccountsTlsVersion.js'),
         'storageAccountHasTags'         : require(__dirname + '/plugins/azure/storageaccounts/storageAccountHasTags.js'),
+        'infrastructureEncryptionEnabled': require(__dirname + '/plugins/azure/storageaccounts/infrastructureEncryptionEnabled.js'),
```

Comment:
change plugin name to infrastructureEncryption
```suggestion
        'infrastructureEncryptionEnabled': require(__dirname + '/plugins/azure/storageaccounts/infrastructureEncryptionEnabled.js'),
```